--- libsvm-3.32/python/setup.py.orig	2024-02-24 18:40:05.425381049 +0100
+++ libsvm-3.32/python/setup.py	2024-02-24 18:41:02.285073013 +0100
@@ -89,9 +89,6 @@ class CleanCommand(clean_cmd):
                 rmtree(f)
 
 def main():
-    if not path.exists(cpp_dir):
-        create_cpp_source()
-
     if not path.exists(license_file):
         copyfile(license_source, license_file)
 
@@ -110,11 +107,6 @@ def main():
         url="https://www.csie.ntu.edu.tw/~cjlin/libsvm",
         license=license_name,
         install_requires=["scipy"],
-        ext_modules=[
-            Extension(
-                "{}.{}".format(PACKAGE_DIR, dynamic_lib_name), **kwargs_for_extension
-            )
-        ],
         cmdclass={"clean": CleanCommand},
     )
 
--- libsvm-3.36/python/libsvm/svm.py.orig	2025-05-12 06:29:51.000000000 +0200
+++ libsvm-3.36/python/libsvm/svm.py	2025-07-05 20:40:08.941084638 +0200
@@ -22,10 +22,7 @@ __all__ = ['libsvm', 'svm_problem', 'svm
             'PRINT_STRING_FUN', 'kernel_names', 'c_double', 'svm_model']
 
 try:
-    dirname = path.dirname(path.abspath(__file__))
-    dynamic_lib_name = 'clib.cp*'
-    path_to_so = glob(path.join(dirname, dynamic_lib_name))[0]
-    libsvm = CDLL(path_to_so)
+    libsvm = CDLL("libsvm.so.4")
 except:
     try:
         if sys.platform == 'win32':
